stock_picking_type_shipping_policy/models/stock_picking_type.py

```diff
@@ -19,6 +19,7 @@ def _selection_shipping_policy(self):
         ]
 
     shipping_policy = fields.Selection(
+        string="Force Shipping Policy",
```

Comment:
Changed the user's label to not conflict with `move_type`/`Shipping Policy` std field

Reply:
Done:

- previous `shipping_policy` field is replaced by `force_move_type` boolean field
- migration script added to set the std field `move_type` based on the value of previous `shipping_policy` + set `force_move_type` to True
- remove override of `stock.move` and switched to an override of `<stock.picking>._compute_move_type`
- DESCRIPTION of the module updated (BTW the mentioned module `stock_routing_operation` doesn't exist, I supposed it should be `stock_dynamic_routing`?)

Reply:
Should we rename the module now that Odoo has some kind of support of the shipping policy on operation types? Like `stock_picking_type_force_move_type` or `stock_picking_type_force_shipping_policy`?